pkg/operators/filter/filter.go

```diff
@@ -426,3 +426,5 @@ func getFilterFunc(f datasource.FieldAccessor, op comparisonType, negate bool, s
 func init() {
 	operators.RegisterDataOperator(&filterOperator{})
 }
+
+var FilterOperator = &filterOperator{}
```

Comment:
For users which consume IG as a Golang library, and willing to use the filter operator, they would need to pass a pointer to the operator to the `gadgetcontext.WithDataOperators` option. To enable this behavior, we need to export the filter operator in some way. I chose global variable, since ocihanelder and clioperator use this way too, so I wanted to stay consistent with the coding style of the go library from the consumers standpoint.

Reply:
Done